netebpfext/net_ebpf_ext_bind.c

```diff
@@ -126,6 +126,11 @@ _net_ebpf_extension_bind_on_client_attach(
         (net_ebpf_extension_hook_client_t*)attaching_client, filter_context);
 
 Exit:
+    if (result != EBPF_SUCCESS) {
```

Comment:
Maybe call `net_ebpf_extension_wfp_filter_context_cleanup` instead?

Reply:
Yes we can call `net_ebpf_extension_wfp_filter_context_cleanup`, but all other instances (in files sock_addr.c, sock_ops.c, etc.) are directly calling `ExFreePool`, so I used the same here for parity.